Doc/library/sqlite3.rst

```diff
@@ -406,14 +406,23 @@ Connection Objects
 
    .. attribute:: isolation_level
 
-      Get or set the current default isolation level. :const:`None` for autocommit mode or
-      one of "DEFERRED", "IMMEDIATE" or "EXCLUSIVE". See section
-      :ref:`sqlite3-controlling-transactions` for a more detailed explanation.
+      This attribute controls the :ref:`transaction handling
+      <sqlite3-controlling-transactions>` performed by ``sqlite3``.
+      If set to :const:`None`, transactions are never implicitly opened.
+      If set to one of ``"DEFERRED"``, ``"IMMEDIATE"``, or ``"EXCLUSIVE"``,
+      the strings representing the underlying SQLite `transaction behaviour`_,
```

Comment:
```suggestion
      corresponding to the underlying `SQLite transaction behaviour`_,
```

Clearer/cleaner phrasing and include SQLite in the link target, since it is a key part of the target's name

Reply:
+1